src/wrappers.hpp

```diff
@@ -42,13 +42,18 @@ namespace flexiblesusy {
 // Constants ///////////////////////////////////////////////////////////
 
 static constexpr double Pi             = 3.141592653589793;
+static constexpr double oneOver16Pi    = 1./(16.*Pi);
```

Comment:
To ensure that this number has full precision, I'd recommend to write:
~~~.cpp
static constexpr double oneOver16Pi    = 0.019894367886486917; // 1/(16 Pi)
~~~